### DISCUSSION TITLE
TrainingPeaks and Training Templates

### DISCUSSION THREAD

**Discussion by fguisso** [zepp-health/zeppos-forum#46] [Q&A]
Hi, I'm using TrainingPeaks with T Rex 2 and have two features to implement:
- Sync my planned workout with Zepp App Training Templates
  - TrainingPeaks can generate a .fit ou .json file with a planned workout, but I need an endpoint or something to import this file to Zepp Training Templates.
- Sync my workout result from Zepp to TrainingPeaks
  - Today I can import the Zepp workout .gpx to TrainingPeaks, but it's done manually, a sync like Strava is the best way.

Questions:
- Is there an API for the Zepp app with an `/import_training_template`?
- How can I create or automate something to access my workout results and export them directly for another app?

**Comment by MyoungXUE** [comment]
Hi, sorry for the long wait, I just finished confirming with my colleagues.

To answer the first question, we do not support this import method for now.

To answer the second question, TrainingPeaks has temporarily closed the third-party API service, so it can't be automatically synchronized like Strava.

**Reply by fguisso** [reply]
Okay, figure out more options in these questions:
1˚ - Can you explain better how Training Templates work? Maybe it's a Zepp App that lives just inside the watch?
2˚ - There is a Zepp endpoint to export the workout? Not the better way, but with this, I can automate something.
